doc/level1.rst

```diff
@@ -249,40 +249,55 @@ Internal state storage
 See :ref:`peer-state` for a definition of the structure of
 information stored about the client's communications peers.
 
-.. todo::
-
-   Explain why we keep peer state.  because we want to handle weird cases like the following...
-
-If a remote peer disables Autocrypt or drops back to using a
-non-Autocrypt MUA only we must be able to disable sending encrypted
-mails to this peer automatically.  MUAs capable of Autocrypt Level 1
-therefore MUST store state about the capabilities of their remote
-peers.
+Autocrypt clients keep state about their peers, to be able to handle
+several nuanced situations that have caused trouble/annoyance in the
+past.  This state is updated even when the peer does *not* send mail
```

Comment:
Well, it is updated when they send mail with no Autocrypt header. It is not updated when they don't send mail at all.